en/Home.md

```diff
@@ -25,11 +25,11 @@ Deepin is a Linux distribution dedicated to providing beautiful, easy to use, se
 
 # Our Community
 
-- [Community Manifesto *Value of the deepin community.*](/about_deepin/deepin_community)
-- [Code of conduct *Guidelines for the deepin community.*](/about_deepin/deepin_community)
-- [SIG & Teams: *special interest groups and other organization  etc.*](/about_deepin/deepin_community)
-- [International communities *Collection of links to the deepin Linux communities around the world.*](/about_deepin/deepin_community)
-- [Mailing list: *mailing list of community.*](/about_deepin/deepin_community)
+- [Community Manifesto *Values of the Deepin community.*](/about_deepin/deepin_community)
```

Comment:
deepin's branding guideline actually suggest use `deepin` (lowercase `d`) instead of `Deepin` when it was used standalone.